### DISCUSSION TITLE
Can't open Artifacts screen

### DISCUSSION THREAD

**Discussion by seelenverwandter1** [community/community#118241] [Mobile] [answered]
Its not possible to open the Artifacts screen in the Andorid App.
It you click on like "2 Artifacts" its just reopening that run screen. https://github.com/community/community/assets/7594455/f81cd144-a22f-4c71-8860-5fafc5dc7364

**Comment by alcere** [comment, marked as answer]
Hi @seelenverwandter1! Thank you for reporting this issue. I was able to reproduce the problem and the team will look into it as soon as possible. Please stay tuned for updates! Thanks again!
